lib/cute_pets/pet_fetcher.rb

```diff
@@ -129,10 +133,18 @@ def get_petfinder_shelter_id
   end
 
   def get_petharbor_shelter_id
-    get_shelter_id(ENV.fetch('petharbor_shelter_id'))
+    ENV.fetch('petharbor_shelter_id')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.